sqldelight-compiler/src/main/kotlin/com/squareup/sqldelight/util/NameAllocatorUtil.kt

```diff
@@ -0,0 +1,11 @@
+package com.squareup.sqldelight.util
+
+import com.squareup.javapoet.NameAllocator
+
+internal fun NameAllocator.getOrSet(objRef: Any, name: String): String {
```

Comment:
I also added a workaround like this to one of my projects. How do you guys feel about adding a method like this to `NameAllocator`?